keyboards/planck/rev6/rules.mk

```diff
@@ -31,9 +30,10 @@ API_SYSEX_ENABLE = no
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 #SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
 
-CUSTOM_MATRIX = yes # Custom matrix file
+CUSTOM_MATRIX = no # Custom matrix file
```

Comment:
```suggestion
```

Reply:
... that's what I was forgetting!   I meant to do that, but ....